runtime/proto/corfu_options.proto

```diff
@@ -19,6 +19,8 @@ message SchemaOptions {
     repeated string stream_tag = 5;
     // Nested Secondary Key (repeated field)
     repeated SecondaryIndex secondary_key = 6;
+    // Logical group name that helps club a number of tables into one group for Log Replication.
+    optional string logical_group = 7;
```

Comment:
Should we include the client name as well?
Also I am wondering if we should have a flag as well (like isFederated) for logical groups use case.
My thought is that the design would then be uniform: a flag to indicate which replication model to use and then followed by the model specific options.
The flags can be an enum.
It can make the dynamicStreams part, find streams for a model, also consistent.

Reply:
Done, added `client_name`
But for the other enum I am open to suggestions but my understanding is the current `is_federated` automatically implies a `FULL_TABLE` replication model (since we need to support the original active standby use case as well) and the new `logical_group` specifies the new use cases for LogicalGroup?
The reason for not having an enum is that it implies that a single table can only be in one model and right now it is not clear if the same table will need to be replicated from active to standby in one deployment and from source to sink in SaaS deployment?

So to keep it simple - just adding 2 new fields - `client_name` and `logical_group`
Does that sound ok?

Reply:
My understand was also that if `logical_group` field is set, it implies that the table is used for logical_group model and we can group the streams to replicate by its value. Adding another enum for this could introduce more work for clients to maintaining them I think? but definitely open to suggestions...

Reply:
I also think the enum would add more complexity, while at the same time, we'd need to do some upgrade management for legacy 'is_federated' flag. Same as Chris explained I also see the presence of option ' logical _group' implicitly indicating the replication model for the given table.

Shama if we're missing some potential benefit of an enumeration please add it. 

Also, let's define wether the same table can belong or not to different replication models, cause if it can't we should include this in our linter.

Reply:
I agree that it will implicitly indicate the model, but think that the flag will make looking for streams uniform in the code. We don't need to look at specific options itself just the flag.
Both will get the work done, but I think flags will be easy to maintain/read.

I am not seeing the complexity right now apart from the upgrade management..
But if everyone agrees on having the "logical-group" directly, I'm okay too. My reason isn't very strong.